Dockerfile

```diff
@@ -0,0 +1,16 @@
+# This Dockerfile is intended for use by openshift/ci-operator config files defined
+# in openshift/release for v4.x prow based PR CI jobs
+
+# NOTE: our openshift jenkins maven agent image was unfeasible since it has a non-root user
+# set and we can't create the ~/.m2 repository dirs
```

Comment:
why couldn't you run "USER 0" to set the user back to root to do what you need?

Reply:
I thought we were not allowed to do that from an openshift docker strategy build ... or am I mis-remebering ?

certainly doing that when I ran `docker build ...` from the command line worked

Reply:
you're misremembering.  

s2i builder images are not allowed to run with user 0 by default.  We don't enforce anything about what your Dockerfile does in a docker strategy build.

Reply:
ah gotcha thanks ... ok, I'll modify all 3 plugins to leverage our maven image and nuke the yum install piece

Reply:
ok update pushed @bparees ptal